infra/cifuzz/cifuzz.py

```diff
@@ -203,6 +209,40 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   return True, False
 
 
+def check_fuzzer_build(out_dir):
+  """Checks the integrity of the built fuzzers.
+
+  Args:
+    out_dir: The location of the fuzzer binaries.
+
+  Returns:
+    True if fuzzers are correct or False if not.
```

Comment:
I think the style I usually see is to leave the "False if not" part implicit.

Reply:
Removed.